File: app/src/main/java/dev/ragnarok/fenrir/api/model/longpoll/VkApiGroupLongpollUpdates.java
package dev.ragnarok.fenrir.api.model.longpoll;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public final class VkApiGroupLongpollUpdates {

    @SerializedName("failed")
    public int failed;

    @Nullable
    @SerializedName("ts")
    public String ts;

    public int getCount() {
        return 0;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/longpoll/VkApiGroupLongpollUpdates.kt
package dev.ragnarok.fenrir.api.model.longpoll

import com.google.gson.annotations.SerializedName

class VkApiGroupLongpollUpdates {
    @SerializedName("failed")
    var failed = 0

    @SerializedName("ts")
    var ts: String? = null
    val count: Int
        get() = 0
}